doc/ADDB-User-Guide.md

```diff
@@ -0,0 +1,79 @@
+# ADDB User Guide
```

Comment:
From the content of this document, looks like it is not a guide on how to use ADDB to instrument Motr or its applications, or a guide on how to parse and analyse ADDB records. IMO, something like "How-to analyse IO Operation using ADDB" may be a better title.